src/pheval_exomiser/runner.py

```diff
@@ -22,6 +29,67 @@ class ExomiserPhEvalRunner(PhEvalRunner):
     config_file: Path
     version: str
 
+    def install(self, overwrite_existing: bool = False):
+        # Parse configuration options
+        configuration = ExomiserConfigurations.parse_obj(
+            self.input_dir_config.tool_specific_configuration_options
+        )
+        phenotype_version = configuration.application_properties.phenotype_data_version
+
+        # Define filenames and URLs
+        base_url = "https://data.monarchinitiative.org/exomiser"
+        filenames = [
+            f"{phenotype_version}_hg19.zip",
+            f"{phenotype_version}_hg38.zip",
+            f"{phenotype_version}_phenotype.zip",
+        ]
+        urls = [
+            f"{base_url}/{self.version}/exomiser-cli-{self.version}-distribution.zip",
+            *[f"{base_url}/data/{filename}" for filename in filenames],
+        ]
+
+        # Download files
+        os.makedirs("./data/tmp/", exist_ok=True)
```

Comment:
Please use `pathlib` for consistency - that way we can also remove the additional import